app/about/layout.tsx

```diff
@@ -31,10 +31,6 @@ export default async function AboutPageBaseLayout (props: AboutPageBaseLayoutPro
 
   const dehydratedState = dehydrate(queryClient);
 
-  if (process.env.NEXT_PUBLIC_ABOUT_PAGE_ENABLED !== "true") {
-    return notFound();
-  }
```

Comment:
I removed `NEXT_PUBLIC_ABOUT_PAGE_ENABLED` and just turned it on in production. I figured we're okay with this given that the copy is has now been looked over by Mairead and Harriet and isn't my placeholder copy anymore and that we're going to start linking people here pretty soon.